requirements

```diff
@@ -1 +1,2 @@
 django>=1.4,<1.9
+ua_parser>=0.7.1
```

Comment:
Should pin this if you want this particular version. As written it'll still install 1.0.X or whatever you didn't want to bother with.